src/components/src/main/java/org/apache/jmeter/extractor/json/render/RenderAsJmesPathRenderer.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.jmeter.extractor.json.render;
+
+import org.apache.jmeter.extractor.json.jmespath.JMESPathCache;
+import org.apache.jmeter.util.JMeterUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+/**
+ * Implement ResultsRender for JMES Path tester
+ * @since 5.2
+ */
+public class RenderAsJmesPathRenderer extends AbstractRenderAsJsonRenderer {
+    private static final Logger log = LoggerFactory.getLogger(RenderAsJmesPathRenderer.class);
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+    @Override
+    protected String getTabLabel() {
+        return JMeterUtils.getResString("jmespath_tester_title");
+    }
+
+    @Override
+    protected String getTestButtonLabel() {
+        return JMeterUtils.getResString("jmespath_tester_button_test");
+    }
+
+    @Override
+    protected String getExpressionLabel() {
+        return JMeterUtils.getResString("jmespath_tester_field");
+    }
+
+    @Override
+    protected String process(String textToParse) {
+        String expression = getExpression();
+        try {
+            JsonNode actualObj = OBJECT_MAPPER.readValue(textToParse, JsonNode.class);
+            JsonNode result = JMESPathCache.getInstance().get(expression).search(actualObj);
+            if (result.isNull()) {
+                return NO_MATCH; //$NON-NLS-1$
+            } else {
```

Comment:
Please refrain from adding `else` branches after `return`.

Often it makes the code harder to read.